taxi/views.py

```diff
@@ -1,5 +1,8 @@
+from audioop import reverse
```

Comment:
what is it and do you use it?

Reply:
I think that it was acidently added, when I tried to use reverse func, and I imported it with using helper, and didn't saw where from It was mported